### PR TITLE
data: SwapnilPatle01

### PR DESCRIPTION
**Pull request by SwapnilPatle01**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4192"></a>

### CODE REVIEW

**Review by stevenovak123 — DISMISSED** [review]
LGTM. Thanks you for contributing

**Comment by loftwah** [comment]
The JSON looks good to me.

**Review by iamthecloverly — APPROVED** [review]

File: data/SwapnilPatle01.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer,Youtuber",
+  "avatar": "https://github.com/SwapnilPatle01.png",
```
Comment (Pradumnasaraf) [inline data/SwapnilPatle01.json:6]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that. If you need any help feel free to ask.

Updated docs: https://linkfree.eddiehub.io/docs/quickstart

**Pull request closed without merging**